go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/ava-labs/awm-relayer
 go 1.22.6
 
 require (
-	github.com/ava-labs/avalanchego v1.11.11-0.20240813203340-ab83fb41528d
-	github.com/ava-labs/coreth v0.13.8-fixed-genesis-upgrade.0.20240813194342-7635a96aa180
-	github.com/ava-labs/subnet-evm v0.6.9-0.20240816202746-18633729a0cd
-	github.com/ava-labs/teleporter v1.0.6
+	github.com/ava-labs/avalanchego v1.11.11
+	github.com/ava-labs/coreth v0.13.8-fixed-genesis-upgrade.0.20240815193440-a96bc921e732
+	github.com/ava-labs/subnet-evm v0.6.9
+	github.com/ava-labs/teleporter v1.0.7-0.20240906195543-f23da2da8fdd
```

Comment:
We should use a tagged version of `teleporter`. You can create a tag yourself whenever you like.

We should also update the submodule to be the same tag. It's slightly hard to find now, but lives at `tests/contracts/lib/teleporter/`